windows/setup_py2exe.py

```diff
@@ -0,0 +1,296 @@
+        #raw_input()
+
+def assert_module(module):
+    try:
+        imp.find_module( module )
+    except ImportError:
+        logger.error('Failed to import '+ module)
+        logger.error('Process aborted because of error!')
+        sys.exit(1)
+
+def run_cmd(cmd):
+    logger.info(cmd)
+    p = subprocess.Popen(cmd, stdin=subprocess.PIPE, stdout=subprocess.PIPE, stderr=subprocess.PIPE )
+    stdout, stderr = p.communicate()
+    logger.info(stdout)
+    if stderr :
+        logger.error(stderr)
+
+
+logger.info('Getting BleachBit version')
+import bleachbit.Common
+BB_VER = bleachbit.Common.APP_VERSION
+logger.info('BleachBit version ' + BB_VER )
+
+
+logger.info('Checking for translations')
+assert_exist('locale', 'run "make -C po local" to build translations')
+
+logger.info('Checking for GTK')
+assert_exist(GTK_DIR)
+
+logger.info('Checking PyGTK+ library')
+assert_module('pygtk')
+
+logger.info('Checking Python win32 library')
+assert_module('win32file')
+
+logger.info('Checking for UPX')
+assert_exist(UPX_EXE, 'Download UPX executable from http://upx.sourceforge.net/')
+
+logger.info('Checking for CodeSign.bat')
+check_exist('CodeSign.bat', 'Code signing is not available')
+
+logger.info('Checking for NSIS')
+check_exist(NSIS_EXE, 'NSIS executable not found: will try to build portable BleachBit')
+
+
+#if not fast:
+#    logger.info('echo Pre-compressing executables')
+#    python_install_path =  os.path.dirname( sys.executable )
+#
+#    python_files = recursive_glob(python_install_path, ['*.pyd'])
+#    for file in python_files:
+#        compress(UPX_EXE, UPX_OPTS, file)
+#
+#    gtk_files = recursive_glob(GTK_DIR, ['*.exe', '*.dll'])
+#    for file in gtk_files:
+#        compress(UPX_EXE, UPX_OPTS, file)
+#
+#    pywintypes27 = python_install_path + '\\Lib\\site-packages\\pywin32_system32\\pywintypes27.dll'
+#    compress(UPX_EXE, UPX_OPTS, pywintypes27)
+
+logger.info('Deleting directories build and dist')
+shutil.rmtree('build', ignore_errors=True)
+shutil.rmtree('dist', ignore_errors=True)
+
+logger.info('Running py2exe')
+shutil.copyfile( 'bleachbit.py', 'bleachbit_console.py')
+cmd= sys.executable +  ' -OO setup.py py2exe'
+run_cmd(cmd)
+assert_exist('dist\\bleachbit.exe')
+assert_exist('dist\\bleachbit_console.exe')
+os.remove('bleachbit_console.py')
+
+
+if not os.path.exists('dist'):
+    os.makedirs('dist')
+
+logger.info('Copying GTK files and icon')
+shutil.copyfile( GTK_DIR + '\\bin\\intl.dll',  'dist\\intl.dll')
+
+shutil.copytree(GTK_DIR + '\\etc', 'dist\\etc')
+shutil.copytree(GTK_DIR + '\\lib', 'dist\\lib')
+shutil.copytree(GTK_DIR + '\\share', 'dist\\share')
+shutil.copyfile( 'bleachbit.png',  'dist\\share\\bleachbit.png')
+
+
+#logger.info('Compressing executables')
+#files = recursive_glob('dist', ['*.pyd', '*.dll', '*.exe'])
+#for file in files:
+#    compress(UPX_EXE, UPX_OPTS, file)
+
+
+logger.info('Purging unnecessary GTK+ files')
+cmd= sys.executable +  ' setup.py clean-dist'
```

Comment:
Regarding the log like this
````warning: msgunfmt missing: skipping recompile````
https://ci.appveyor.com/project/az0/bleachbit/build/1.13.285#L798

Here would you please enable `msgunfmt` to work?